miso-web/src/main/webapp/styles/room-6WT.html

```diff
@@ -24,22 +23,69 @@
   <body>
     <svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" width="200mm" height="487.9476mm" viewBox="0 0 200 487.9476" version="1.1">
       <g transform="translate(296.4873,-35.185715)">
-        <rect id="004744" width="19.455463" height="10.159981" x="-173.5211" y="43.1553" style="fill:#ff0000"/>
-        <rect style="fill:#0000ff" id="004746" width="11.42327" height="11.150295" x="-197.55641" y="43.155296" ry="0"/>
-        <rect style="fill:#000000" id="004919" width="10.627469" height="10.995018" x="-219.58118" y="43.155296" ry="0"/>
-        <rect style="fill:#0000ff" id="004747" width="10.338972" height="11.150295" x="-208.42455" y="43.155296"/>
-        <rect style="fill:#ff0000" id="004504" width="9.4578018" height="10.430261" x="-173.52473" y="65.745041"/>
-        <rect style="fill:#0000ff" id="004750" width="8.9201202" height="10.456921" x="-183.41371" y="65.678223"/>
-        <rect style="fill:#0000ff" id="003555" width="9.5548859" height="10.423512" x="-193.90404" y="65.711632"/>
-        <rect style="fill:#ffff00" id="003701" width="15.100456" height="9.2678232" x="-211.0761" y="66.900726"/>
-        <rect style="fill:#ff0000" id="004152" width="7.8500834" height="7.2829614" x="-157.68901" y="137.37341"/>
-        <rect style="fill:#ffff00" id="003697" width="7.8500834" height="7.2829614" x="-166.59741" y="137.37341"/>
-        <rect style="fill:#ff0000" id="003558" width="11.666953" height="10.373226" x="-219.49716" y="256.89893"/>
-        <rect style="fill:#ff0000" id="004733" width="9.3045692" height="8.1365519" x="-220.02629" y="307.57327"/>
-        <rect style="fill:#000000" id="004781" width="12.239555" height="11.961703" x="-219.74849" y="460.60822"/>
-        <rect style="fill:#ff0000" id="001200" width="12.239556" height="11.961703" x="-206.92215" y="460.60165"/>
-        <rect style="fill:#ff0000" id="003559" width="12.239557" height="11.961703" x="-166.63713" y="460.60165"/>
-        <rect style="fill:#008000" id="003700" width="12.15742" height="8.4788733" x="-219.69759" y="127.42441"/>
+        <a href="/miso/freezer/barcode/004744" onmouseout="document.getElementById('label').textContent = ''" onmouseover="document.getElementById('label').textContent = '004744'">
```

Comment:
Could the onmouseover text for this room be updated to also include the island name, like in the 6ST diagram?

Reply:
Nope. They haven't yet picked island names for those freezers, so I will update that at a later date.